core/tests/protractor_utils/AdminPage.js

```diff
@@ -209,8 +208,11 @@ var AdminPage = function() {
   this._stopOneOffJob = async function(jobName, i) {
     var text = await (await unfinishedOneOffJobRows.get(i)).getText();
     if (text.toLowerCase().startsWith(jobName.toLowerCase())) {
-      await (await unfinishedOneOffJobRows.get(i)).element(
-        by.css('.protractor-test-one-off-jobs-stop-btn')).click();
+      var unfinishedOffJobRowsButton = (
+        await unfinishedOneOffJobRows.get(i)).element(
+        by.css('.protractor-test-one-off-jobs-stop-btn'));
```

Comment:
Indentation here is off:

```suggestion
      var unfinishedOffJobRowsButton = (
        await unfinishedOneOffJobRows.get(i)
      ).element(
        by.css('.protractor-test-one-off-jobs-stop-btn'));
```

Also like before, the `await` is unnecessary